packages/datadog-plugin-cucumber/src/index.js

```diff
@@ -0,0 +1,95 @@
+const { relative } = require('path')
+
+const { SAMPLING_RULE_DECISION } = require('../../dd-trace/src/constants')
+
+const {
+  TEST_TYPE,
+  TEST_NAME,
+  TEST_SUITE,
+  TEST_STATUS,
+  getTestEnvironmentMetadata
+} = require('../../dd-trace/src/plugins/util/test')
+
+function setStatusFromResult (span, result) {
+  if (result.status === 1) {
+    span.setTag(TEST_STATUS, 'pass')
+  } else if (result.status === 2) {
+    span.setTag(TEST_STATUS, 'skip')
+    span.setTag('error.msg', 'skipped')
+  } else if (result.status === 4) {
+    span.setTag(TEST_STATUS, 'skip')
+    span.setTag('error.msg', 'not implemented')
+  } else {
+    span.setTag(TEST_STATUS, 'fail')
+    span.setTag('error.msg', result.message)
+  }
+}
+
+function createWrapRun (tracer, testEnvironmentMetadata, sourceRoot) {
+  return function wrapRun (run) {
+    return async function handleRun () {
```

Comment:
`async` is not needed here since the only await in the function is also not necessary.